package.json

```diff
@@ -5,7 +5,7 @@
   "main": "knex.js",
   "types": "types/index.d.ts",
   "engines": {
-    "node": ">=8"
+    "node": ">=10"
```

Comment:
travis config file needs to be updated not to run tests on Node 8 as well

Reply:
ok